site/users/index.html.js

```diff
@@ -48,6 +48,16 @@ var logos = [
     img: 'reindex.png',
     link: 'https://www.reindex.io/'
   },
+  {
+    name: 'Meteor',
+    img: 'meteor.png',
+    link: 'https://medium.com/apollo-stack/snappier-uis-with-apollo-client-graphql-bffab0b52b79#.sls7hdvud'
```

Comment:
Linking to the blog post series is a good idea :) - it's nice if these links go somewhere that actually tells more of the story rather than just a link to the company site, so that's cool!